observation_tickets/Observation_ticket_creator.pyw

```diff
@@ -52,6 +56,103 @@ def quit_func():
     savetxt()
     master.quit()
 
+def check_toi():
+    '''
+    Description
+    -----------
+    Checks to see if a target has been selected, and if so,
+    displays it on the widget
+
+    '''
+    global info_directory
+    link = 'https://docs.google.com/spreadsheets/d/17tqqRnaXuI9q_w1uz35wttQQFGpOIP3hhVXZ5-bYnQA/export?format=csv&id=17tqqRnaXuI9q_w1uz35wttQQFGpOIP3hhVXZ5-bYnQA'
+    savefile = requests.get(url=link)
+    current_directory = os.path.abspath(os.path.dirname(__file__))
+    info_directory = os.path.join(current_directory, r'toi_info')
+    try:
+        os.mkdir(info_directory)
+    except:
+        pass
```

Comment:
I'd replace this with a simple check, the less try/catch statements, the better:
if os.path.exists(info_directory):
  os.makedirs(info_directory)